consts/others.go

```diff
@@ -1,11 +1,12 @@
 package consts
 
 const (
-	StrBasicAuthPrefix              string = "Basic "
-	StrTokenAuthPrefix              string = "Auth Token "
-	StrEmailTokenVerificationPrefix string = "Email Token "
-	StrBasicAuthAttempt             string = "Basic Auth Attempt"
-	StrTokenAuthAttempt             string = "Token Auth Attempt"
-	StrMdBasicAuthHeader            string = "authorization"
-	StrMdAuthToken                  string = "auth_token"
+	StrBasicAuthPrefix               string = "Basic "
+	StrTokenAuthPrefix               string = "Auth Token "
+	StrEmailTokenVerificationPrefix  string = "Email Token "
+	StrBasicAuthAttempt              string = "Basic Auth Attempt"
+	StrAuthTokenAttempt              string = "Auth Token Attempt"
+	StrEmailTokenVerificationAttempt string = "Email Token Verification Attempt"
+	StrMdBasicAuthHeader             string = "authorization"
+	StrMdAuthToken                   string = "auth_token"
```

Comment:
The accepted header for authentication/authorization and email verification from the frontend are the following:
- `"authorization": "Email Token " + <token>`
- `"authorization": "Basic " + <base64 encoded email:password>`
- `"authorization": "Auth Token " + <token_string>`